index.html

```diff
@@ -0,0 +1,124 @@
+              </li>
+              <li class="navItem">
+                <a class="navLink" href="#" aria-disabled="true">Sponsor</a>
+              </li>
+              <li class="navItem">
+                <a class="navLink" href="#" aria-disabled="true">News</a>
+              </li>
+            </ul>
+            <div id="campaign" class="campaign"><a class="campaignLink" href="#"> Campaign</a></div>
+          </div>
+        </div>
+    </nav>
+
+    <section class="headline flex-column">
+        <div class="introductionText">
+            <h1>"Hello! Teaching world"<span>EDUCATIONAL ROBOTIC MEETING IN AFRICA 2023</span></h1>
+        </div>
+        <div class="descriptionText">
+            <p>A keynote talk on the impact of robotics and AI on the future of education is taking place in Cameroon. Discover the potential benefits of educational robotics in enhancing students' problem-solving and critical-thinking skills and preparing them for future careers.</p>
+        </div>
+        <div class="dateAndPlace">
+            <h1>2023.04.25(TUE) ~ 26(WED)</h1>
+            <p>@ National Advanced School of Engineering, Yaounde</p>
+        </div> 
+    </section>
+
+    <section class="mainProgram flex-column">
+      <h2>Main Program</h2>
+      <div class="underlineMainTitle"></div>
+      <div class="mainProgramItemsBox flex-column">
+        <div class="mainProgramItem flex-row">
+          <img class="mainProgramImage" src="/icons/lecture.png" alt="lecture">
+          <h3>Lecture</h3>
+          <p>Didactic approaches in educational robotics.</p>
+        </div>
+        <div class="mainProgramItem flex-row">
+          <img class="mainProgramImage" src="/icons/exhibition.png" alt="exhibition">
+          <h3>Expo.</h3>
+          <p>Appreciate various creations and projects made by students.</p>
+        </div>
+        <div class="mainProgramItem flex-row">
+          <img class="mainProgramImage" src="/icons/forum.png" alt="forum">
+          <h3>forum</h3>
+          <p>Have a time to share you thoughts and opinions with experts.</p>
+        </div>
+        <div class="mainProgramItem flex-row">
+          <img class="mainProgramImage" src="/icons/workshop.png" alt="workshop">
+          <h3>Workshop</h3>
+          <p>Try to create your own robot under the supervision of an expert. </p>
+        </div>
+        <div class="mainProgramItem flex-row">
+          <img class="mainProgramImage" src="/icons/networking.png" alt="networking">
+          <h3>Networking</h3>
+          <p>Get opportunity to network with experts.</p>
+        </div>
+      </div>
+      <div class="joinButton"><a class="joinLink" href="Join ">Join Educational Robotics Meeting 2023</a></div>
+    </section>
+
+    <section id="featuredSpeakers" class="featuredSpeakers flex-column">
+      <h2>Featured Speakers</h2>
+      <div class="underlineTitle"></div>
+      <div id="featuredSpeakersBox" class="featuredSpeakersBox"></div>
```

Comment:
- [ ] Each speaker is an article containing six individual articles, each with an image, title, subtitle, and description. Instead of using a div here kindly use the `article` tag for semantic purposes